deploy/shopping-store.yaml

```diff
@@ -0,0 +1,10 @@
+# Deployment spec for a store. A store is a logical abstraction over a database
+# deployment, and captures how to deploy and connect to that database. Multiple
+# StatefulServices's would use a single store.
```

Comment:
I think this is not a possessive form? And would be StatefulServices I think.

Reply:
Yep, you're right! I copied that from the original JS sample, so it needs to be fixed there as well.